nautobot/extras/tests/test_api.py

```diff
@@ -547,16 +536,15 @@ def test_create_object_required_fields(self):
         self.assertHttpStatus(response, status.HTTP_400_BAD_REQUEST)
         self.maxDiff = None
         self.assertEqual(
-            response.data,
+            str(response.data["label"][0]),
```

Comment:
It's not clear what index `0` is here.

Reply:
It is the drf list of errors `[ErrorDetail(string='This field is required.', code='required')]`. So we are basically getting the first and only item and casting it to an str object.

Reply:
Revert the changes